.buildkite/test-pipeline.yaml

```diff
@@ -50,219 +62,257 @@ steps:
   - VLLM_ATTENTION_BACKEND=XFORMERS pytest -v -s basic_correctness/test_chunked_prefill.py
   - VLLM_ATTENTION_BACKEND=FLASH_ATTN pytest -v -s basic_correctness/test_chunked_prefill.py
   - VLLM_TEST_ENABLE_ARTIFICIAL_PREEMPT=1 pytest -v -s basic_correctness/test_preemption.py
-
-- label: Core Test
+  
+- label: Core Test # 10min
   mirror_hardwares: [amd]
   fast_check: true
+  source_file_dependencies:
```

Comment:
Besides the source file dependencies, I think we should add the tests themselves as dependency too. Like if `tests/core/` changes, this test should also run. Maybe we can rename `source_file_dependencies` to `dependencies` to be inclusive?